pilot/utils/model_utils.py

```diff
@@ -2,10 +2,12 @@
 
 
 def _clear_torch_cache(device="cuda"):
+    try:
+        import torch
+    except ImportError:
+        return
```

Comment:
why this can clean torch cache?

Reply:
Now we can use DB-GPT without installing pytorch when we use proxy LLM and embedding model, see [feat(core): Multi-module dependency splitting](https://github.com/eosphoros-ai/DB-GPT/pull/606) for details. This is a supplement to the previous [PR](https://github.com/eosphoros-ai/DB-GPT/pull/606)

Reply:
I understand, but this function then should not name to `_clear_torch_cache` IMO

Reply:
Aha, I use a new function `_clear_model_cache` to clean the cache.